data/schema.graphql

```diff
@@ -2396,6 +2396,24 @@ type CreditCard {
 
   # Credit card's expiration year
   expiration_year: Int!
+
+  # Billing address street1
+  street1: String
```

Comment:
It's a shame that these don't match our other address forms.

Reply:
yeah totally agreed 😢